telecom-db/telecom.sql

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+import pandas as pd
+conn = sqlite3.connect("telecom.db")
+cur = conn.cursor()
+
+table_query = cur.execute("SELECT name FROM sqlite_master WHERE type='table'")
+tables = table_query.fetchall()
+
+joined = cur.execute("""
+            
+            SELECT t.*, et.event_type, lf.log_feature, lf.volume, st.severity_type 
```

Comment:
Left out resource_type table and column. Also FYI: the solution uses LEFT JOIN if you want to try that to see if it gives you the same output.